cmd/populator/cmd/register/register.go

```diff
@@ -13,6 +13,7 @@ func NewRegister(cliName string) *cobra.Command {
 
 	hub.AddCommand(
 		NewCapactInstallation(cliName),
+		NewTestStorageBackend(cliName),
```

Comment:
As the test storage backend installation is separate from E2E tests, I think it is good to register it once someone decided it should be installed. Because if we want to have a test setup, let's have it fully all at once. In that way:

1. We can also use this storage also for development purposes
2. We are consistent with the way how we register TypeInstances for whole Capact installation (and the test storage is a part of Capact installation)

Registering this TI during tests would definitely make sense, but if we install the test storage backend during test run.

Reply:
Make sense. :+1: